src/Akeneo/Pim/Enrichment/Component/Product/Normalizer/Standard/Product/ProductValueNormalizer.php

```diff
@@ -128,4 +126,34 @@ protected function getSimpleValue(ValueInterface $value, ?string $format = null,
 
         return $this->normalizer->normalize($value->getData(), $format, $context);
     }
+
+    /**
+     * Cut all the ending zeros after coma if decimals are more than 4
+     * @return int|string
+     */
+    private function formatNumber(ValueInterface $value, AttributeInterface $attribute)
+    {
+        if (!$attribute->isDecimalsAllowed()) {
+            return (int)$value->getData();
+        }
+
+        if (strpos($value->getData(), '.') === false) {
+            return $value->getData();
+        }
+
+        $dataWithoutZero = rtrim($value->getData(), '0');
+        $dataWithoutZero = rtrim($dataWithoutZero, '.');
+
+        if (strpos($value->getData(), '.') === false) {
```

Comment:
```suggestion
        if (strpos($dataWithoutZero, '.') === false) {
```

Sorry I gave you a wrong solution